lib/ui/profile/ProfileListItem.dart

```diff
@@ -144,4 +154,27 @@ class ProfileListItem extends StatelessWidget {
       overflow: TextOverflow.ellipsis,
     );
   }
+
+  void _actionTapped(BuildContext context) {
+    if (_viewModel.isDestructive) {
+      Alert.present(
+        Alert(
+          viewModel: AlertViewModel(
+            cancelActionText: _LocalisedStrings.cancelAction(),
```

Comment:
I'm not sure about that but I was wondering about having actions inside view models.
In some point of chat integration we have decided that logic as onTap should to outside the View Model and be part of widget inputs. In this case we have confirmAction which is a Function. Should it be part of view model?

Reply:
It depends on what we are doing. if the action is mutating to model layer then this should be done via the view model. if its mutating the view layer (moving to another screen) then probably not. In this case, maybe the alert should not have a view model at all, or we rename it to something else like "setup". since I'm just reusing it here in this PR, I think its something to debate later.